### PR TITLE
add html and css to people.html

### PR DESCRIPTION
**Pull request by saugusth**

### CODE REVIEW

**Review by ilopez5 — CHANGES_REQUESTED** [review]
Looks good so far, I have a few suggestions/requests:
1. Maybe add a border radius to the images, so that they are circles
2. The lines within a team section doesn't look that good
3. Add a section for Past Contributors (get a list from Dr. Dan)
4. If you aren't set on the placeholder image, maybe a generic silhouette would look good? I leave the final decision up to the website team.

File: assets/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.style-fix img {
```
Comment (sahil-at-home1) [inline assets/css/styles.css:1]: I think we should try to use bootstrap as much as possible to simplify our code, while keeping it fluid and responsive. Bootstrap includes HTML classes to setup images, along with borders, and different shapes (like circles). I noticed that the images are different sizes, which might make it difficult for bootstrap to align them alone. In that case or some other bootstrap limitation, I support this.

File: people.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                    <p>Chris <br> Core <br> VR </p>
+                </div>
+            </div>
+            <div style="background-color:black; border: 1px solid;"></div>
+            <div class="row row-cols-4">
+                <!--col-lg-3 means that on large scrrens, colmn will take up fourth the length (12/3 = 4).
+                col-md-4 means that on medium screens, colmn will take up a third of the length (12/4 = 3).  
+                col-sm-6 means that on small screens, colmn will take up half the of the length (12/6 = 2).-->
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/Julia.jpeg" alt="Place holder">
```
Comment (sahil-at-home1) [inline people.html:56]: small typo, should be `.jpg`

File: people.html [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/circuit_board.png" alt="Place holder">
+                    <p>Sahil <br> Software-Website</p>
+                </div>
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/circuit_board.png" alt="Place holder">
+                    <p>Son <br>Software-Website</p>
+                </div>
+            </div>
+            <div style="background-color:black; border: 1px solid;"></div>
```
Comment (sahil-at-home1) [inline people.html:72]: I think its better to minimize usage of inline CSS, but this is small so I am open to keeping it. If we wanted to use bootstrap we could use something like `<hr></hr>` or `<div class="divider"></div>`

**Review by sahil-at-home1 — CHANGES_REQUESTED** [review]
Overall it looks really good. Nice job on using the Bootstrap grid here. I left some comments, but overall I think that maybe the images should be smaller. I also think that we can leave the subgroups and titles out (like website, VR, etc.), and possibly even leave the groups out (software, hardware), and just list people involved with the IPRO per semester. We could possibly add a section in the beginning focusing on Dr. Dan with a short bio.

**Pull request merged**